projects/docs-tools/src/modules/code-examples/code-examples-editor.service.spec.ts

```diff
@@ -187,12 +187,7 @@ describe('Code examples editor service', () => {
   let service: SkyDocsCodeExamplesEditorService;
 
   beforeEach(() => {
-    moduleImports = [
-      'BrowserAnimationsModule',
-      'FormsModule',
-      'ReactiveFormsModule',
-      'RouterModule.forRoot([])',
-    ];
+    moduleImports = ['FormsModule', 'ReactiveFormsModule'];
```

Comment:
I've removed `AppModule` and now bootstrap `AppComponent` as standalone. Theme and router move to `main.ts` and module imports move to `AppComponent`.